tests/acceptance/features/apiGraph/addUserToGroup.feature

```diff
@@ -276,7 +276,7 @@ Feature: add users to group
     Given these groups have been created:
       | groupname | comment      |
       | sales     | normal group |
-    And the user "Admin" has disabled user "Alice" using the Graph API
+    And the user "Admin" has disabled user "Alice"
```

Comment:
Refactored as we discussed, we are not using texts like `using the Graph API` for a given step.